composer.json

```diff
@@ -150,5 +150,10 @@
             "composer phpunit",
             "composer phpstan"
         ]
+    },
+    "config": {
```

Comment:
Could you remove this? To make sure it doesn't get added again, rebase or merge main into your branch, then run `composer up` once more.